content/docs/command-reference/exp/save.md

```diff
@@ -14,33 +20,47 @@ usage: dvc exp save [-h] [-q | -v] [-f]
 
 ## Description
 
-Provides a way to save the current status of your <abbr>project</abbr> as an
-<abbr>experiment</abbr> without polluting it with unnecessary commits, branches,
+Saves a snapshot of your <abbr>project</abbr> as an <abbr>experiment</abbr>,
+without polluting your Git repository with unnecessary commits, branches,
 directories, etc.
 
-Only files tracked by either Git or DVC are saved to the experiment. Use the
-`--include-untracked` (`-I`) option to explicitly include any untracked files in
-the experiment.
-
-`dvc exp save` creates <abbr>experiments</abbr> just like
-[`dvc exp run`](/doc/command-reference/exp/run), but does not require
-reproduction or setup of [pipeline stages](/doc/start/data-pipelines), making it
-possible to quickly start tracking, [comparing] and [persisting] experiments.
+This lets you start tracking, [comparing], [plotting], and [sharing] experiments
+quickly after making any project changes (e.g. retrain an ML model).
 
 [comparing]: /doc/user-guide/experiment-management/comparing-experiments
-[persisting]: /doc/user-guide/experiment-management/persisting-experiments
+[plotting]: /doc/user-guide/experiment-management/visualizing-plots
+[sharing]: /doc/user-guide/experiment-management/persisting-experiments
+
+<admon type="tip">
+
+Only files tracked by either Git or DVC are saved to the experiment unless you
+use `--include-untracked` (`-I`) on untracked files explicitly (see an
+[example](#examples) below).
+
+</admon>
+
+[Review] your experiments with `dvc exp show`. Successful ones can be [made
+persistent] by restoring them via `dvc exp branch` or `dvc exp apply` and
+committing them to the Git repo. Unnecessary ones can be [cleared] with
+`dvc exp gc`.
+
+[review]: /doc/user-guide/experiment-management/comparing-experiments
+[made persistent]: /doc/user-guide/experiment-management/persisting-experiments
+[cleared]: /doc/user-guide/experiment-management/cleaning-experiments
 
 ## Options
 
 - `-n <name>`, `--name <name>` - specify a [unique name] for this experiment. A
   default one will be generated otherwise, such as `exp-f80g4` (based on the
   experiment's hash).
 
-- `-I <path>`, `--include-untracked <path>` - specify untracked file(s) to be
-  included in the saved experiment. Multiple files can be specified.
+- `-I <path>`, `--include-untracked <path>` - specify an untracked file or
+  directory to be included in the experiment. This option can be used multiple
+  times.
 
-- `-f`, `--force` - overwrite the experiment if an experiment with the same name
-  already exists.
+- `-f`, `--force` - rewrite the experiment if it already exists. Or, if nothing
+  has changed in the workspace, force saving an experiment anyway (identical to
+  Git `HEAD`).
```

Comment:
> if nothing has changed in the workspace, force saving an experiment anyway

I guess this applies to `exp run` as well?

Reply:
`exp run` also forces reproduction by passing the `--force` arg over to `repro`

Reply:
@dtrifiro It looks like the core help still needs to be updated after https://github.com/iterative/dvc/pull/8690?

> Or, if nothing
  has changed in the workspace, force saving an experiment anyway (identical to
  Git `HEAD`).

@dtrifiro I missed this check in https://github.com/iterative/dvc/pull/8690. I don't think `--force` should be needed and it complicates this option. Can we drop it?

Ideally, we want to make `dvc exp run` the same and not worry about creating duplicate experiments (see the discussion in https://github.com/iterative/dvc/pull/8659#issuecomment-1338499979).

Reply:
Since iterative/dvc#8743 has been merged:

```suggestion
- `-f`, `--force` - rewrite the experiment if it already exists.
```